sql/item_mods.sql

```diff
@@ -21567,9 +21562,6 @@ INSERT INTO `item_mods` VALUES (19716,27,16);
 INSERT INTO `item_mods` VALUES (19716,161,-16);
 INSERT INTO `item_mods` VALUES (19716,288,25);
 INSERT INTO `item_mods` VALUES (19716,302,5);
-INSERT INTO `item_mods` VALUES (19717,30,30);

```

Comment:
Yes please, keep the format/sort intact. I've found it helps avoid certain types of user error when multiple editors go at this file.